MtoM fake/InformationVC.swift

```diff
@@ -186,53 +192,60 @@ class InformationVC: UIViewController, UITableViewDataSource, UITableViewDelegat
     func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int{
         return CellType.All.rawValue
     }
-    
+    var count :Int = 0
```

Comment:
Cai nay chi de cho em hieu ve the nao la reuse cell thoi. Em hieu chua? Hieu roi thi xoa no di